api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetUpRecencyHTSClientCardDataExportReport2019.java

```diff
@@ -117,6 +118,7 @@ public ReportDefinition constructReportDefinition() {
 		rd.setDescription(getDescription());
 		rd.setParameters(getParameters());
 		rd.addDataSetDefinition("HTSCARDDATAEXPORT", Mapped.mapStraightThrough(dataSetDefinition()));
+		rd.addDataSetDefinition("HTSCARDDATAEXPORT", Mapped.mapStraightThrough(CommonDatasetLibrary.settings()));
```

Comment:
@ODORA0 have you run this? I am not sure it will work. Doesn't Mapped.mapStraightThrough allow multiple definitions?

Reply:
@ssmusoke Not sure about that though. So this would have both the `HTSCARDDATAEXPORT` and `S`?

Reply:
@ssmusoke The dhis2_uuid works well actually adding HTSCARDDATAEXPORT.